CONTRIBUTING.md

```diff
@@ -2,7 +2,7 @@ Hi (future) collaborator!
 
 **tl;dr;**
 - use pull requests
-- use [conventional changelog](https://github.com/ajoslin/conventional-changelog/blob/master/conventions/angular.md) commit style messages
+- use [conventional changelog](https://github.com/conventional-changelog/conventional-changelog) commit style messages
```

Comment:
should be <https://github.com/conventional-changelog/conventional-changelog/blob/master/packages/conventional-changelog-angular/convention.md> to link to the actual convention